tests/benchmark.py

```diff
@@ -83,10 +79,34 @@ def func(tracer, level=0):
 
         # do some work
         num = random.randint(1, 10)
-        span.set_tag('num', num)
+        span.set_tag("num", num)
 
         if level < 10:
             func(tracer, level + 1)
             func(tracer, level + 1)
 
     benchmark(func, tracer)
+
+
+def test_tracer_start_span(benchmark, tracer):
+    benchmark(tracer.start_span, "benchmark")
+
+
+@pytest.mark.benchmark(group="span-id", min_time=0.005)
+def test_span_id_randbits(benchmark):
+    from ddtrace.compat import getrandbits
```

Comment:
if we go the route of no fallback, we probably don't even need these benchmarks here, probably just want one around `_rand.rand64bits()` to track historical changes over time to that code

Reply:
Yep good point. :+1: